src/game/server/entities/character.cpp

```diff
@@ -729,6 +718,19 @@ bool CCharacter::TakeDamage(vec2 Force, int Dmg, int From, int Weapon)
 		m_Health -= Dmg;
 	}
 
+	// create healthmod indicator
+	if(Server()->Tick() < m_DamageTakenTick+25)
+	{
+		// make sure that the damage indicators doesn't group together
+		GameServer()->CreateDamageInd(m_Pos, m_DamageTaken*0.25f, Dmg);
```

Comment:
Creates and sends 2 events right now. We should either use the new event or just send the old one.

Reply:
Right now the server decides the position, angle, etc. of the stars, so we need this kind of event, unless the whole damageind code is ported to the client.

If we only send the old one, this means any graphical effect designed in 0.7.2 will not work with 0.7.1 servers. I'm fine with that if you think that is better than sending currently unused packets, you tell me (code-wise this means removing the CreateDamage server calls for now).

Reply:
Might be best to drop the old CreateDamageInd and just use the new event.

Reply:
Did just that.